internal/connector/ssh_test.go

```diff
@@ -47,13 +46,13 @@ func TestUpdateLocalUsers(t *testing.T) {
 	actual, err := os.ReadFile(logFile)
 	assert.NilError(t, err)
 
-	expected := `pkill '--signal' 'KILL' '--uid' 'three333'
-pkill '--signal' 'KILL' '--uid' 'four444'
-userdel '--remove' 'three333'
-userdel '--remove' 'four444'
-useradd '--comment' 'Ej,managed by infra' '-m' '-p' '*' '-g' 'infra-users' 'two222'
+	expected := `pkill --signal KILL --uid three333
+pkill --signal KILL --uid four444
+userdel --remove three333
+userdel --remove four444
+useradd --comment "Ej,managed by infra" -m -p * -g infra-users two222
```

Comment:
I changed it to only quote the args when there is some whitespace in the arg, so the expected values also changed.

Reply:
It might be better to keep the quotes for consistency. Otherwise, it's up to the author to figure out if/when to wrap expected values in quotes.

Reply:
By author do you mean the person changing this expected value?

You can automatically update this expected value by running tests with `-update`.

I think it reads a bit better without the extra quotes, but it's not a big problem to add them back.

What do you think about adding a comment informing the reader about `-update` flag? Would that work instead of always adding quotes?

Reply:
Yes, author as in author of a commit. I agree this reads better and easier to map to the actual command being written. Don't have to add the quote back in though it might be a good ideal to quote the `*`

Reply:
Good call, I've added a comment about how to update this value, and I've added single quotes for any values with spaces or `*`.